lib/Echidna/UI.hs

```diff
@@ -247,20 +241,19 @@ ui vm world dict initialCorpus cliSelectedContract cs = do
   workerStates workers =
     forM workers $ \(_, stateRef) -> readIORef stateRef
 
-#ifdef INTERACTIVE_UI
  -- | Order the workers to stop immediately
 stopWorkers :: MonadIO m => [(ThreadId, IORef WorkerState)] -> m ()
 stopWorkers workers =
   forM_ workers $ \(threadId, workerStateRef) -> do
     workerState <- readIORef workerStateRef
     liftIO $ mapM_ killThread (threadId : workerState.runningThreads)
 
-vtyConfig :: IO Config
+vtyConfig :: IO VtyUserConfig
 vtyConfig = do
-  config <- Vty.standardIOConfig
-  pure config { inputMap = (Nothing, "\ESC[6;2~", EvKey KPageDown [MShift]) :
-                           (Nothing, "\ESC[5;2~", EvKey KPageUp [MShift]) :
-                           inputMap defaultConfig }
+  pure defaultConfig { configInputMap = [
```

Comment:
an alternative is to use:

```
config <- Vty.userConfig
```

which is empty config or load some user settings. Per doc, I have never tried though!